### PR TITLE
chore: apply dark mode to color settings menu

### PR DESCRIPTION
**Pull request by ericjim**
#### Changes
* Color menu is now updated to reflect theme changes.

#### Closes
* https://github.com/rangle/augury/issues/459

### CODE REVIEW

File: src/frontend/components/header/header.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h4 class="regular flex flex-center primary-color m0">
   </header>
 
   <form>
-    <ul class="setting-dropdown list-reset border border-dark rounded bg-silver absolute right-0" *ngIf="settingOpened">
+    <ul class="setting-dropdown list-reset border border-dark rounded bg-silver absolute right-0" [ngClass]="{'setting-dropdown--dark':theme==='dark'}" *ngIf="settingOpened">
```
Comment (ericjim) [inline src/frontend/components/header/header.html]: There are many ways of add/removing a css class in ng2, I like this one the best. With that said, I am open to other ways of achieving the same result.

**Comment by igor-ka** [comment]
@ericjim LGTM :shipit:

File: src/frontend/components/header/header.html (hunk 1 above)
Comment (sumitarora) [inline src/frontend/components/header/header.html]: @ericjim this is the way we have done it in application for black theme 👍

**Comment by sumitarora** [comment]
:shipit:

**Pull request merged**